README.md

```diff
@@ -0,0 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Reviewing-trial-project
+
+> Objective: Review of a personal pervious work, using the point of view a Code Reviewer
+> Follow and understand the code reviewing process.
+> This project was inspired by the original version done by _Jorge Polo_, my coding partner. You can see his version [here](https://github.com/jpol01/Reviewing-trial-project)
+
+![screenshot](./survey-model.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Html
+- Css
+- GitHub
+
+## Live Demo
+
+[Live Demo Link](https://joshuaivie.github.io/review-trial-project/)
```

Comment:
Here is the link to the live version of your project. You may need to update to a functional link!